subt/docker/sln_bash/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM osrf/subt-virtual-testbed:subt_solution_latest
 
-RUN /usr/bin/python3 -m pip install --user --no-cache-dir msgpack pyserial "opencv-python>3.4.2,<4" pyzmq
```

Comment:
why are you removing preinstalled packages?

Reply:
just `pyserial` - it is not removal - diff just moved the corresponding line down bellow